packs/feats/ricochet-legend.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>The more obstacles between you and your target, the deadlier your shots become, as you ricochet off multiple surfaces to hit them. When using Ricochet Shot, your shot can bounce off of any surface within your weapon's first three range increments and the target can't benefit from cover unless they have total cover on all sides.</p>\n<p>In addition, a creature must attempt a Will save against your class DC the first time in an encounter you attack it with a Ricochet Shot. If it fails, it's @UUID[Compendium.pf2e.conditionitems.Item.Stunned]{Stunned 2} and is unable to determine where your shot originated.</p>"
+            "value": "<p>The more obstacles between you and your target, the deadlier your shots become, as you ricochet off multiple surfaces to hit them. When using Ricochet Shot, your shot can bounce off of any surface within your weapon's first three range increments and the target can't benefit from cover unless they have total cover on all sides.</p>\n<p>In addition, a creature must attempt a @Check[type:will|dc:resolve(@actor.attributes.classDC.value)] save against your class DC the first time in an encounter you attack it with a Ricochet Shot. If it fails, it's @UUID[Compendium.pf2e.conditionitems.Item.Stunned]{Stunned 2} and is unable to determine where your shot originated.</p>"
```

Comment:
This may need to be `@actor.system.proficiencies.classDCs.gunslinger.dc`

Reply:
Looks like both work but the more specific one is probably better in case of multiclass characters? Unlikely with a level 20 feat, but who knows.